Add mLUKE (#14570)
huggingface/transformers: docs/source/model_doc/mluke.rst

..
    Copyright 2021 The HuggingFace Team. All rights reserved.

    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
    the License. You may obtain a copy of the License at

        http://www.apache.org/licenses/LICENSE-2.0

    Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
    an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
    specific language governing permissions and limitations under the License.

mLUKE
-----------------------------------------------------------------------------------------------------------------------

Overview
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The mLUKE model was proposed in `mLUKE: The Power of Entity Representations in Multilingual Pretrained Language Models

> <https://arxiv.org/abs/2110.08151>`_ by Ryokan Ri, Ikuya Yamada, and Yoshimasa Tsuruoka. It's a multilingual extension

Reviewer: ```suggestion
<https://arxiv.org/abs/2110.08151>`__ by Ryokan Ri, Ikuya Yamada, and Yoshimasa Tsuruoka. It's a multilingual extension
```

> of the `LUKE model <https://arxiv.org/abs/2010.01057>`__ trained on the basis of XLMRoBERTa.

Reviewer: ```suggestion
of the `LUKE model <https://arxiv.org/abs/2010.01057>`__ trained on the basis of XLM-RoBERTa.
```

> It is based on XLMRoBERTa and adds entity embeddings, which helps improve performance on various downstream tasks

Reviewer: ```suggestion
It is based on XLM-RoBERTa and adds entity embeddings, which helps improve performance on various downstream tasks
```
